client/src/styling.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from "@emotion/react";
+import { space } from "@guardian/source-foundations";
+
+export const bottom = 108;
+export const right = 15;
+export const floatySize = 40;
+export const boxShadow =
+  "rgba(0, 0, 0, 0.14) 0px 0px 4px, rgba(0, 0, 0, 0.28) 0px 4px 8px";
+export const standardFloatyContainerCss = css`
```

Comment:
this refers to the scrollable panel right?
```suggestion
export const standardPanelContainerCss = css`
```

Reply:
included in #82